examples/ex1.cpp

```diff
@@ -207,12 +211,20 @@ int main(int argc, char *argv[])
       umf_solver.Mult(B, X);
 #endif
    }
-   else // Jacobi preconditioning in partial assembly mode
+   else
    {
       if (UsesTensorBasis(fespace))
       {
-         OperatorJacobiSmoother M(a, ess_tdof_list);
-         PCG(*A, M, B, X, 1, 400, 1e-12, 0.0);
+         if (algebraic_ceed)
```

Comment:
Do we only want to use `algebraic_ceed` when we use tensor basis? The logic was different before.

Reply:
Yes, it only works with tensor basis. The logic in `ex1` is now the same as in `ex1p` (which was correct before).